### PR TITLE
fix(cache): add cache error catching

### PR DESCRIPTION
**Pull request by ya-woodcutter**

### CODE REVIEW

**Comment by ya-woodcutter** [comment]
UP!

File: src/cache.js [hunk 1]
```diff
@@ -41,7 +46,7 @@ export default function cacheMw(settings = {}) {
 
         const cacheKey = createCacheKey(resourceId, method, params);
 
-        const cachedResult = await cache.get(bin, cacheKey);
+        const cachedResult = await cache.get(bin, cacheKey).catch(errorHandler);
```
Comment (evilj0e) [inline src/cache.js:49]: I think you should make errorHandler configurable. Maybe someone want to send errors of that type to sentry / whatever.

Comment (ya-woodcutter) [reply src/cache.js:49]: 👍

**Review by evilj0e — APPROVED** [review]

**Comment by ya-woodcutter** [comment]
continue in https://github.com/auru/unity-api-mw/pull/24

**Pull request closed without merging**